browser/nunjucks-slim.js

```diff
@@ -1,4 +1,4 @@
-/*! Browser bundle of nunjucks 2.3.0 (slim, only works with precompiled templates) */
+/*! Browser bundle of nunjucks 3.0.0-dev.1 (slim, only works with precompiled templates) */
```

Comment:
Please revert the changes to the browserfiles; that's done only at release time, otherwise it causes too many conflicts.